### PR TITLE
How to use the tutorial

### PR DESCRIPTION
**Pull request by triptec**
All work has been done by @MeaningOfLights here (https://github.com/graphile/postgraphile/pull/1577)
I just saw the PR and though is was a small push needed to get it over the finish line.

If there's more to be done say so and I'll try to fix it.

### CODE REVIEW

**Comment by triptec** [comment]
@MeaningOfLights FYI I made the PR against the main repo if there is more to be done.

File: examples/forum/TUTORIAL.md [hunk 1]
```diff
@@ -805,8 +811,180 @@ postgraphile \
...
+*If you get a  **function crypt(text, text) does not exist** error after trying to authenticate, set the Schema for the crypt call: forum-example.crypt in the authenticate function.*
+  
+
+**Using a Json Web Token (JWT)**
+
+In SQL we can see the JWT contains the role and the person_id:
+
+![Alt text](images/6_role.jpg?raw=true "Role is contained in the JWT")
+
+Remember when Graphophile accepts an Authorization Request Header with a valid JWT containing a role field...
```
Comment (jemgillam) [inline examples/forum/TUTORIAL.md:975]: This should be "PostGraphile"

**Comment by jemgillam** [comment]
Thank you, this looks good and easy to read overall. I have yet to actually follow the steps given in the tutorial, but assuming I can then we can get this merged.

One thing is, I think "intellisense" is a term belonging to VS Code's implementation of autocomplete. I think we should use "autocomplete" or "code completion" instead.

File: examples/forum/TUTORIAL.md (hunk 1 above)
Comment (jemgillam) [inline examples/forum/TUTORIAL.md:975]:
```suggestion
Remember when PostGraphile accepts an Authorization Request Header with a valid JWT containing a role field...
```

**Comment by MeaningOfLights** [comment]
Thanks all for your additions. This guide should help share the goodness. My grammar isn't the best and these edits are welcome.

**Comment by jemgillam** [comment]
I've just run it through prettier and then we can merge it, thank you for your work on this :)

**Pull request merged**